examples/pipelines/windows10-installer/README.md

```diff
@@ -0,0 +1,122 @@
+# Windows 10 Installer Pipeline
+
+Downloads a Windows Source ISO into a DV and installs Windows into a new base DV.
+It will then spin up an installation VM and use Windows Answer Files to automatically install the VM.
+Then the pipeline will wait for the installation to complete and will delete the installation VM while keeping the artifact DV with the installed operating system.
+The pipeline can be customized to support different installation requirements.
+
```

Comment:
This pipeline installs Windows into a new DataVolume. This DataVolume is suitable to be used as a default boot source or golden image for Windows VMs.

The pipeline implements this by spinning up a new VM which boots from the Windows disc image (ISO file). The installation of windows is automatically executed and controlled by a Windows answer file. Then the pipeline will wait for the installation to complete and will delete the installation VM while keeping the resulting DataVolume with the installed operating system. The pipeline can be customized to support different installation requirements.